packages/discovery-provider/src/api/v1/users.py

```diff
@@ -1998,14 +1998,28 @@ def get(self):
             ns.abort(400, "JWT payload could not be decoded.")
 
         wallet_user_id = get_user_with_wallet(wallet)
-        if not wallet_user_id or (
-            wallet_user_id != payload["userId"]
-            and wallet_user_id != decode_string_id(payload["userId"])
-        ):
-            ns.abort(
-                404,
-                "The JWT signature is invalid - the wallet does not match the user.",
+        jwt_user_id = decode_string_id(payload["userId"])
+
+        if not wallet_user_id:
+            ns.abort(404, "The JWT signature is invalid - invalid wallet")
+
+        # If the user id found in the token does not match the signer of the token,
+        # check if the signing user is a manager of that user. Otherwise, reject.
+        if wallet_user_id != jwt_user_id:
```

Comment:
is [`is_active_manager`](https://github.com/AudiusProject/audius-protocol/blob/3f133aad3a57927330d8e7e3d2faa0d543343d15/packages/discovery-provider/src/queries/get_managed_users.py#L161) or [`check_authorized`](https://github.com/AudiusProject/audius-protocol/blob/3f133aad3a57927330d8e7e3d2faa0d543343d15/packages/discovery-provider/src/api/v1/access_helpers.py#L10) sufficient for this? Just don't want to duplicate if we don't need to.

If they aren't, should update line 2011 here to also require `is_revoked=False`

Reply:
TIL! great method `is_active_manager`!